docs/articles/commands/custom_context_checks.md

```diff
@@ -23,8 +23,11 @@ public class DirectMessageUsageAttribute : ContextCheckAttribute
 
 This is the attribute that you will apply to the command method.
 
-## Implementing the Check Method
-Now we're going to implement the logic which checks if the command is allowed to be executed. The `IContextCheck<T>` interface is used to define the check method. The `T` is the attribute/parameters that's associated with the command. In this case, it's the `DirectMessageUsageAttribute`.
+## Implementing the context check
+Now we're going to implement the logic which checks if the command is allowed to be executed. The interface `IContextCheck<T>` is used to define the check method. The `T` is the attribute that was applied to the command. In this case, it's the `DirectMessageUsageAttribute`, but it can be any check attribute - if desired, there can be multiple checks for one attribute.
```

Comment:
*multiple checks per attribute

Reply:
> The interface `IContextCheck<T>`

This does not read well to me, the other way around is right.